RAW/ViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    var posts : [PostModel] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        fetchDataWithAPI()
+    }
+    
+    func fetchDataWithAPI(){
+        WebService().fetchRawData(sucess:{[weak self](userData) in
+            DispatchQueue.main.async {[weak self] () -> Void in
+                guard let `self` = self else {return}
+                self.posts = userData
+                self.tableView.reloadData()
+            }
+            }, failure:{[weak self](error) in
+                guard let `self` = self else {return}
+                let alert = UIAlertController(title:"Alert", message:error, preferredStyle:.alert)
+                alert.addAction(UIAlertAction.init(title:"ok", style:.cancel, handler:nil))
+                self.present(alert, animated: true, completion: nil)
+        })
+    }
+    
+}
+
+extension ViewController: UITableViewDelegate, UITableViewDataSource{
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return posts.count
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    fileprivate func updateCelldata(_ indexPath: IndexPath, _ cell: UITableViewCell) {
```

Comment:
why move it in a different method? It's a single cell, so it will be more readable in the method itself.

Reply:
Was doing refactoring.